avalanchego-conformance/README.md

```diff
@@ -1,3 +1,8 @@
+# avalanche-conformance 
+
+avalanche-conformance is a tool used to ensure code is compatible with avalanchego. The tool performs byte level checking for the avalanchego codec as well as key generations. The code is run against a live avalanchego node. 
```

Comment:
can you elaborate on what "code" you are referring to here?

Reply:
I'll do a deep dive on this crate and provide more details.